src/main/kotlin/nl/info/zac/zaak/exception/CaseWithADecisionCannotBeTerminatedException.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ *
+ */
+
+package nl.info.zac.zaak.exception
+
+class CaseWithADecisionCannotBeTerminatedException(message: String) : RuntimeException(message)
```

Comment:
```suggestion
class ZaakWithADecisionCannotBeTerminatedException(message: String) : RuntimeException(message)
```

We should be careful not to translate 'zaak' by 'case' in our code base because the word 'case' really means something else in the wider world, and even in our ZAC context. A case a CMMN term, a zaak is really something else. It is a ZGW term. Technically speaking a zaak consists of, amongst others, a case, but it is not the same thing. In our code base we agreed that we will not translate the word `zaak` for this reason.